bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataGrpcClientWrapper.java

```diff
@@ -0,0 +1,65 @@
+package com.google.cloud.bigtable.grpc;
+
+import com.google.api.core.ApiFuture;
+import com.google.bigtable.v2.MutateRowRequest;
+import com.google.cloud.bigtable.config.BigtableOptions;
+import com.google.cloud.bigtable.core.IBigtableDataClient;
+import com.google.cloud.bigtable.core.IBulkMutation;
+import com.google.cloud.bigtable.data.v2.internal.RequestContext;
+import com.google.cloud.bigtable.data.v2.models.ConditionalRowMutation;
+import com.google.cloud.bigtable.data.v2.models.InstanceName;
+import com.google.cloud.bigtable.data.v2.models.ReadModifyWriteRow;
+import com.google.cloud.bigtable.data.v2.models.Row;
+import com.google.cloud.bigtable.data.v2.models.RowMutation;
+
+public class BigtableDataGrpcClientWrapper implements IBigtableDataClient {
+
+  private BigtableDataGrpcClient bigtableDataGrpcClient;
```

Comment:
Can this be final?  Also, can we use the `BigtableDataClient` instead of `BigtableDataGrpcClient`?  Please call the variable `delegate`.